grunt_tasks/plugin.js

```diff
@@ -369,16 +370,16 @@ module.exports = function (grunt) {
                                 [`girder_plugins/${plugin}/node`]: pluginNodeDir,
                                 [`girder_plugins/${plugin}`]: webClient
```

Comment:
I do think we should remove the first one, since other plugins shouldn't be referencing npm deps of other plugins since that's probably undefined behavior.

However, this might be considered a breaking change, so may involve more conversation.